lib/elephant_in_the_room_web/templates/site/_authors.html.eex

```diff
@@ -0,0 +1,31 @@
+<%
+  alias ElephantInTheRoomWeb.Uploaders.Image
+  authors = assigns[:authors]
```

Comment:
Values in `assigns` can be access with `@`, for example: `@authors`

Reply:
yes, but if not authors is assigned, `@authors` crashes while `assigns[:authors]` returns `nil`